Dockerfile

```diff
@@ -59,6 +59,7 @@ RUN echo "cHJpbnRmICJrcnp5c3p0b2YuZ29yZ29sZXdza2lAZ21haWwuY29tXG41MTcyXG4gKkN2dW
 RUN apt-get update && \
     apt-get install -y --no-install-recommends \
                     fsl-core=5.0.9-1~nd+1+nd16.04+1 \
+                    fsl-atlases=5.0.7-2 \
```

Comment:
I see - this is needed to get the right reference grid! Let's try using `fsl-mni152-templates` to save some space.